services/discovery/discovery.go

```diff
@@ -48,6 +48,28 @@ func FromDependencies(deps resource.Dependencies, name string) (Service, error)
 }
 
 // Service describes the functions that are available to the service.
+//
+// For more information, see the [Discovery service docs].
```

Comment:
[opt] Looks like on other resources this “more info” line and its link are repeated at the top of the file, ie lines 2-4. Seems redundant but maybe add there for consistency for now?